utils/user-product-helper.ts

```diff
@@ -5,7 +5,7 @@ export const getProductInfo = (config, type: ProductCode): ProductI => {
   switch (type) {
     case ProductCode.BUSINESS:
       return {
-        image: 'BCRS_dashboard_thumbnail_image',
+        image: 'BCRS_dashboard_thumbnail_image.jpg',
```

Comment:
By removing the hard-coded jpg extension from the display component, we now support PNGs or other image types.